src/eth/provider.ts

```diff
@@ -1,9 +1,10 @@
-import { connection, ConnectionResponse } from 'decentraland-connect'
-import { WebSocketProvider } from 'eth-connect'
+import type { ConnectionResponse, Provider } from 'decentraland-connect/dist/types'
+import { connection } from 'decentraland-connect/dist/ConnectionManager'
+import { ProviderAdapter } from 'decentraland-connect/dist/ProviderAdapter'
+import { WebSocketProvider } from 'eth-connect/providers/WebSocketProvider'
```

Comment:
Could you move this import up, please? It should be before Decentraland-related packages.

Reply:
`eth-connect` is a decentraland-related package

Reply:
My bad 🙏